samples/highcharts/series/linecap/demo.js

```diff
@@ -0,0 +1,25 @@
+Highcharts.chart('container', {
+
+    plotOptions: {
+        series: {
+            marker: {
+                enabled: false
+            },
+            lineWidth: 15
+        }
+    },
+
+    xAxis: {
+        min: -1,
+        max: 5
+    },
+
+    series: [{
+        data: [2, 5, 2, 3, 6],
+        linecap: 'round' // default
+    }, {
+        data: [3, 6, 3, 4, 7],
+        linecap: 'square'
+    }]
+
+});
```

Comment:
I would personally add this demo to the `highcharts\samples\highcharts\plotoptions` folder

Reply:
The `plotoptions` folder is pretty cramped... So I would prefer adding a new folder for line series on the pattern `series-{{seriestype}}`:

`/samples/highcharts/series-line/linecap`